clarifai/cli/templates/model_templates.py

```diff
@@ -99,7 +99,7 @@ def get_server(self) -> FastMCP:
 '''
 
 
-def get_openai_model_class_template(port: str = "8000") -> str:
+def get_openai_model_class_template(port: str = "8000", model_name: str = "google/gemma-3n-e4b", host: str = "localhost") -> str:
```

Comment:
we don't need these extra args as they aren't used in this template, only the local version of it.